From: Carlos Garnacho <carlosg@gnome.org>
Date: Tue, 17 Oct 2023 12:48:42 +0200
Subject: tracker-extract: Initialize modules also before commandline
 extraction

This allows the gstreamer module to block plugins through the registry
in those paths too.
---
 src/tracker-extract/tracker-main.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/tracker-extract/tracker-main.c b/src/tracker-extract/tracker-main.c
index 209e2a2..3c7ee23 100644
--- a/src/tracker-extract/tracker-main.c
+++ b/src/tracker-extract/tracker-main.c
@@ -327,6 +327,9 @@ do_main (int argc, char *argv[])
 
 	tracker_content_identifier_cache_init ();
 
+	tracker_extract_module_manager_init ();
+	tracker_module_manager_load_modules ();
+
 	/* Set conditions when we use stand alone settings */
 	if (filename) {
 		return run_standalone ();
@@ -338,8 +341,6 @@ do_main (int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
-	tracker_module_manager_load_modules ();
-
 	miner_dbus_name = tracker_domain_ontology_get_domain (domain_ontology,
 	                                                      MINER_FS_NAME_SUFFIX);
 	sparql_connection = tracker_sparql_connection_bus_new (miner_dbus_name,
